DicesGame/Dices.cs

```diff
@@ -28,41 +31,153 @@ public Dices()
+                    PlayerTurnLabel.Text = "Fader's turn to roll!";
+                }
+                counter = 0;
+            }
+            else
+            { 
+                _dices[0].GenerateRandomFace(random1);
+                ShooterDice1.Image = _dices[0].GetImage();
+                _dices[1].GenerateRandomFace(random2);
+                ShooterDice2.Image = _dices[1].GetImage();
+            }
+        }
+
+        private void timerFaderRoll_Tick(object sender, EventArgs e)
+        {
+            counter++;
+            if (counter == 5)
+            {
+                timerFaderRoll.Stop();
+                _dices[2].RollTheDice(random1);
+                _dices[3].RollTheDice(random2);
+                FaderDice1.Image = _dices[2].GetImage();
+                FaderDice2.Image = _dices[3].GetImage();
+                if (_roundNumber % 2 == 0)
+                {
+                    timerHouseAfterFaderRoll.Enabled = true;
+                    PlayerTurnLabel.Text = "House's turn to roll!";
+                }
+                else
+                {
+                    PlayGameButton.Enabled = true;
+                    gameWinner = _gameRule.GetWinner(_dices[0].GetResult() + _dices[1].GetResult(), _dices[2].GetResult() + _dices[3].GetResult());
+                    if (gameWinner == 0)
+                    {
+                        GameWinnerLabel.Text = "Draw";
+                    }
+                    else if (gameWinner == 1)
+                    {
+                        GameWinnerLabel.Text = "Fader wins";
+                        winningsTextbox.Text = (betNumericUpDown.Value * 2).ToString();
+                    }
+                    else
+                    {
+                        GameWinnerLabel.Text = "House wins";
+                        winningsTextbox.Text = "0";
+                    }
+                    _roundNumber++;
+                    ChangeGameRuleButton.Enabled = true;
+                    betNumericUpDown.Enabled = true;
+                }
+                counter = 0;
             }
             else
             {
-                PlayerTurnLabel.Text = "Fader's turn to roll";
+                _dices[2].GenerateRandomFace(random1);
+                FaderDice1.Image = _dices[2].GetImage();
+                _dices[3].GenerateRandomFace(random2);
+                FaderDice2.Image = _dices[3].GetImage();
             }
         }
 
-        private void timer1_Tick(object sender, EventArgs e)
+        private void timerHouseAfterFaderRoll_Tick(object sender, EventArgs e)
         {
             counter++;
-            if (counter == 10)
+            if (counter == 5)
             {
-                _dices[0].RollTheDice();
-                _dices[1].RollTheDice();
-                timer1.Enabled = true;
+                timerHouseAfterFaderRoll.Stop();
+                timerHouseRoll.Enabled = true;
                 counter = 0;
             }
+            
```

Comment:
maybe do something like
if counter != 5 
    return;

//the code from the if block